src/static/order.php

```diff
@@ -22,100 +67,54 @@
                     <a href="login.html">Login</a>
                 </div>
                 -->
-
             </div>
         </div>
     </div>
     <div class="content-wrapper">
        <div class="order-form-wrapper">
-            <h1>Thank you for your order!</h1>
+            <h1>Online Orders</h1>
+            <p>Use the form below to send us your order.</p>
+            
+            <form action="order.php" method="post">
+                <label for="user_id">User ID:</label><br>
+                <input type="text" id="user_id" name="user_id" value="userid#" required><br><br>
+                <label for="email">Enter your email:</label><br>
+                <input type="email" id="email" name="email" required><br><br>
+                <label for="address">Shipping address:</label><br>
+                <input type="text" id="address" name="address" value="123 shark lane Sharksville CO 12345" required><br><br>
+                <label for="itemid">Item ID:</label><br>
+                <input type="text" id="itemid" name="itemid" value="item id#" ><br>
+                <label for="itemquant"> Item quantity:</label><br>
+                <input type="number" id="itemquant" name="itemquant" min="1" ><br><br>
+                <label for="myfile">Or upload an order file:</label><br>
+                <input type="file" id="myfile" name="myfile"><br><br>
+                <input type="submit" value="Submit">
+                <input type="reset">
+              </form>
        </div>
     </div>
+    
+</body> 
+</html>
+
  
   <?php
 
-<<<<<<< HEAD
-  $servername = "cs-3250-database-1testing.ctxpxr8jzoap.us-west-1.rds.amazonaws.com";
-  $username = "admin";
-  $password = "cs3250db1";
-  $dbname = "cs3250main";
-  $tablename = "sales_orders";
-  $maintable = "simsharktable"
-=======
-// Include config file
-require_once "config.php";
-  //$maintable = "shark_table"
->>>>>>> 015259e72b4916d21ed884367523df29162d23d5
 
-  // Check connection
-  if ($link->connect_error) {
-    die("Connection failed: " . $conn->connect_error);
-  }
 
-  $email = filter_input(INPUT_POST, 'email');
-  $useraddress = filter_input(INPUT_POST, 'address');
-  $user = filter_input(INPUT_POST, 'user_id');
-  $productid = filter_input(INPUT_POST, 'itemid');
-  $productquantity = filter_input(INPUT_POST, 'itemquant');
-  $orderdate = time();
-
-  //$sql = "select $maintable"
-
-  //to do: only allow them to order as much as we have
-
-  //add a date field, email, location, fufilled
-  //add fufilled column and date column/time stamp
-  $sql = "INSERT INTO $tablename (product_id, quantity, userID, order_date, fufilled )
-  VALUES ('$productid', '$productquantity', '$user', '$orderdate', 'no')";
- //to do: post price
-
-<<<<<<< HEAD
-  if ($conn->query($sql) === TRUE) {
-=======
- //to do: update shark table with orders
-  if ($link->query($sql) === TRUE) {
->>>>>>> 015259e72b4916d21ed884367523df29162d23d5
-    echo "<b>Your order: </b><br><br>";
```

Comment:
something of this form should be added back to the form. the echoing the order here. or it could be echoed as a confirmation on the thankyou page.